blogposts/2021/release-3.27.md

```diff
@@ -0,0 +1,69 @@
+[Read more about why we made this change](https://about.sourcegraph.com/blog/postgres-version-update/). 
+
+changelogItems:
+  # Added
+  - description: "`count:` now supports \"all\" as value. Queries with `count:all` will return up to 999999 results."
+    url: https://github.com/sourcegraph/sourcegraph/pull/19756
+    category:
+  - description: Credentials for Batch Changes are now validated when adding them.
+    url: https://github.com/sourcegraph/sourcegraph/pull/19602
+    category: Batch Changes
+  - description: "Batch Changes now ignore repositories that contain a `.batchignore` file. [#19877](https://github.com/sourcegraph/sourcegraph/pull/19877) and src-cli#509"
+    url: "https://github.com/sourcegraph/src-cli/pull/509"
+    category: Batch Changes
+  - description: "Side-by-side diff for commit visualization."
+    url: https://github.com/sourcegraph/sourcegraph/pull/19553
+    category:
+  - description: "The site configuration now supports defining batch change rollout windows, which can be used to slow or disable pushing changesets at particular times of day or days of the week. [#19796](https://github.com/sourcegraph/sourcegraph/pull/19796), [#19797](https://github.com/sourcegraph/sourcegraph/pull/19797), and #19951."
+    url: https://github.com/sourcegraph/sourcegraph/pull/19951
+    category: Batch Changes
+# Changed
+  - description: "Bumped the minimum supported version of Postgres from `9.6` to `12`. The upgrade procedure is mostly automated for existing deployments, but may require action if using the single-container deployment or an external database. See the upgrade documentation for your deployment type for detailed instructions."
+    url: https://docs.sourcegraph.com/admin/updates
+    category:
+  - description: "Changesets in batch changes will now be marked as archived instead of being detached when a new batch spec that doesn't include the changesets is applied. Once they're archived users can manually detach them in the UI."
+    url: https://github.com/sourcegraph/sourcegraph/pull/19527
+    category: Batch Changes
+  - description: "The default replica count on `sourcegraph-frontend` and `precise-code-intel-worker` for Kubernetes has changed from `1` -\u003e `2`."
+    url:
+    category:
+  - description: Changes to code monitor trigger search queries
+    url: https://github.com/sourcegraph/sourcegraph/pull/19680
+    category: Search
+  - description: "Deprecated the GraphQL `icon` field on `GenericSearchResultInterface`. It will be removed in a future release."
+    url: https://github.com/sourcegraph/sourcegraph/pull/20028/files
+    category: Search
+  - description: "Creating changesets through Batch Changes as a site-admin without configured Batch Changes credentials has been deprecated. Please configure user or global credentials before Sourcegraph 3.29 to not experience any interruptions in changeset creation."
```

Comment:
```suggestion
  - description: "Creating changesets through Batch Changes as a site-admin without configured Batch Changes credentials has been deprecated. To avoid any interruptions in changeset creation, please configure user or global credentials before Sourcegraph 3.29."
```